app/search.py

```diff
@@ -8,19 +8,52 @@
 import webbrowser
 from googlesearch import search
 import sys
+import os
 
 def open_browser(urls):
     for url in urls:
         webbrowser.open_new_tab(url)
 
-def search_my_term(term, count):
-    return [url for url in search(term, stop=count)]
+def search_my_term(term,s, count):
+    return [url for url in search(term, start=s,stop=count)]
 
+def ignore():
+    with open("data/ignore.txt") as f:
+        return f.read().splitlines()
```

Comment:
You can also use `f.readlines()`

Reply:
but using readlines() would produce a \n at the end ,right?

Reply:
Oh that's right. Good catch!